packages/arb-token-bridge-ui/src/hooks/useNetworksAndSigners.tsx

```diff
@@ -227,62 +244,85 @@ export function NetworksAndSignersProvider(
       return
     }
 
-    // Case 3
-    getL1Network(provider as Web3Provider)
-      .then(async l1Network => {
-        // Web3Provider is connected to an L1 network. We instantiate a provider for the L2 network.
-        const l2Provider = new StaticJsonRpcProvider(
+    // Case 4
+    getParentChain(provider as Web3Provider)
```

Comment:
We are slowly moving away from the idea of L1 and L2. Because now L2 <> L3 transfers are possible, we now have `ParentChain` and `Chain`.